src/components/Onfido/BaseOnfidoWeb.tsx

```diff
@@ -46,7 +57,6 @@ function initializeOnfido({sdkToken, onSuccess, onError, onUserExit, preferredLo
             colorBorderLinkUnderline: theme.link,
             colorBackgroundLinkHover: theme.link,
             colorBackgroundLinkActive: theme.link,
-            authAccentColor: theme.link,
             colorBackgroundInfoPill: theme.link,
             colorBackgroundSelector: theme.appBG,
             colorBackgroundDocTypeButton: theme.success,
```

Comment:
```suggestion
            colorBackgroundDocTypeButton: theme.success,
            borderWidthSurfaceModal: '0px',
```

@VickyStash Could you help to add borderWidthSurfaceModal property to hide the black border (more information [here](https://github.com/Expensify/App/pull/37713#issuecomment-1996893360))